README.md

```diff
@@ -44,6 +44,20 @@ almanack table path/to/repository
 almanack check path/to/repository
 ```
 
+### Pre-commit Hook
+
+We provide [pre-commit](https://pre-commit.com/) hooks to enable you to run the Almanack as part of your automated checks for software projects.
```

Comment:
```suggestion
We provide [pre-commit](https://pre-commit.com/) hooks to enable you to run the Almanack as part of your automated checks for developing software projects.
```

Reply:
Also, is it worth adding any detail about the pre-commit hook around line 9?

Reply:
Thanks, and yes, we should - I'll add that in.